src/explore-education-statistics-admin/src/pages/users/UserInvitePage.tsx

```diff
@@ -91,67 +120,175 @@ const UserInvitePage = ({
+                    legendSize="m"
+                    hint="The user's release roles within the service."
+                  >
+                    <FormFieldSelect
+                      label="Release"
+                      name="selectedReleaseId"
+                      placeholder="Choose release"
+                      options={model?.releases?.map(release => ({
+                        label: release.title,
+                        value: release.id,
+                      }))}
+                    />
+                    <FormFieldSelect
+                      label="Release role"
+                      name="selectedReleaseRole"
+                      placeholder="Choose release role"
+                      options={model?.resourceRoles?.Release?.map(role => ({
+                        label: role,
+                        value: role,
+                      }))}
+                    />
+                    <Button
+                      type="button"
+                      className="govuk-!-margin-top-6"
+                      onClick={() => {
+                        if (
+                          !form.values.selectedUserReleaseRoles.some(
+                            role =>
+                              role.releaseId ===
+                                form.values.selectedReleaseId &&
+                              role.releaseRole ===
+                                form.values.selectedReleaseRole,
+                          )
+                        ) {
+                          const newUserReleaseRole = {
+                            releaseId: form.values.selectedReleaseId,
+                            releaseTitle: model?.releases.filter(
+                              r => r.id === form.values.selectedReleaseId,
+                            )[0].title,
+                            releaseRole: form.values.selectedReleaseRole,
+                          };
+                          form.setFieldValue('selectedUserReleaseRoles', [
+                            ...form.values.selectedUserReleaseRoles,
+                            newUserReleaseRole,
+                          ]);
+                        }
+                      }}
+                    >
+                      Add release role
+                    </Button>
+
+                    {form.values.selectedUserReleaseRoles.length === 0 ? (
+                      <p>No user release roles added</p>
+                    ) : (
+                      <table>
+                        <thead>
+                          <tr>
+                            <th scope="col">Release Title</th>
+                            <th scope="col">Release Role</th>
+                            <th scope="col">Options</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {form.values.selectedUserReleaseRoles.map(
+                            selectedUserReleaseRole => (
```

Comment:
Simplify this to `userReleaseRole` in-line with other suggestion to remove 'selected' prefixes.